### PR TITLE
LEX-965 - Update faraday and faraday_middleware

### PR DESCRIPTION
**Pull request by kenneth1870ga**
**Ticket:** https://generalassembly.atlassian.net/browse/LEX-965 
 Update faraday and faraday_middleware dependencies 

**Tasks**

- [x] Tested Locally

### CODE REVIEW

File: ims-lti.gemspec [hunk 1]
```diff
@@ -20,11 +19,11 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'addressable', '~> 2.5', '>= 2.5.1'
   spec.add_dependency 'builder', '~> 3.2'
-  spec.add_dependency 'faraday', '< 2.0'
-  spec.add_dependency 'faraday_middleware', '< 2.0'
+  spec.add_dependency 'faraday', '~> 1'
+  spec.add_dependency 'faraday_middleware', '~> 1.2'
```
Comment (kevinrood) [inline ims-lti.gemspec:23]: If the dependencies here were previously using `< 2.0`, should we use `< 3.0` for both of these gems now? That might be more in line with what the gem maintainers are expecting.

File: ims-lti.gemspec [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
 
 Gem::Specification.new do |spec|
   spec.name = 'ims-lti'
   spec.version = IMS::LTI::VERSION
   spec.authors = ['Instructure']
   spec.email = 'opensource@instructure.com'
-  spec.summary = %q{Ruby library for creating IMS LTI tool providers and consumers}
-  spec.homepage = %q{http://github.com/instructure/ims-lti}
+  spec.summary = 'Ruby library for creating IMS LTI tool providers and consumers'
+  spec.homepage = 'http://github.com/instructure/ims-lti'
```
Comment (kevinrood) [inline ims-lti.gemspec:12]: Since our goal is to upstream these changes, do you think the maintainer will have any issue with these changes?

**Review by kevinrood — COMMENTED** [review]
@kenneth1870ga 👏 nice work here.

I left you a couple comments. In addition, could you please change the branch to something more descriptive? Once everything is set in this PR, I would like to have you open a PR to the upstream repo and it would be best not to include our ticket number in the branch used for that PR.

**Comment by kenneth1870ga** [comment]
Closed because the branch name was updated

**Pull request closed without merging**